pipelines/Dfe.Spi.Common.Logging.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  paths:
+    include:
+      - /pipelines/Dfe.Spi.Common.Logging
```

Comment:
this should be z`/pipelines/Dfe.Spi.Common.Logging.yml`; my error in the other 2 builds